webfront/tests/tests_entry_endpoint.py

```diff
@@ -187,3 +187,32 @@ def test_can_get_protein_amount_from_interpro_id_pfam_id(self):
             "'proteins' should be one of the keys in the response",
         )
         self.assertEqual(response.data["metadata"]["counters"]["proteins"], 1)
+
+
+class ReplaceTIGRFamsTest(InterproRESTTestCase):
+    def test_can_read_ncbifam(self):
+        urls = [
+            "/api/entry/ncbifam",
+            "/api/entry/ncbifam/nf000004",
+            "/api/protein/uniprot/entry/ncbifam/nf000004",
+        ]
+        for url in urls:
+            response = self.client.get(url)
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_can_redirect_db(self):
+        urls = [
+            "/api/entry/tigrfams",
+            "/api/entry/tigrfams/nf000004",
+            "/api/protein/uniprot/entry/tigrfams/nf000004",
+        ]
+        for url in urls:
+            response = self.client.get(url)
+            self.assertEqual(response.status_code, status.HTTP_302_FOUND)
+            self.assertRedirects(
+                response,
+                url.replace("tigrfams", "NCBIfam"),
```

Comment:
Keeping URL in lower case for consistency:

```suggestion
                url.replace("tigrfams", "ncbifam"),
```